src/solver/coeffs.jl

```diff
@@ -0,0 +1,324 @@
+#  TODO(Alex): it is not clear to me at the moment if we should use
+# term or TermInterface.maketerm
```

Comment:
So `maketerm` does expand `Base.MathConstants.pi` and friends into floating point numbers. 
We can maybe switch to `Symbolics.term`. What do you think ?